api/app/models/spree/webhooks/endpoint.rb

```diff
@@ -0,0 +1,7 @@
+module Spree
+  module Webhooks
+    class Endpoint < Spree::Base
+      self.table_name = 'spree_webhooks_endpoints'
```

Comment:
@sebastian-palma let's create `Spree::Webhooks::Base` class and set `self.table_name_prefix = 'spree_webhooks'` there